CHANGELOG.md

```diff
@@ -1,3 +1,112 @@
+# 1.2.0
+
+1. Breaking change which mostly maintains backwards compatibility, see
+   "Breaking Changes" below.
+2. Support for monadic generators e.g. [mwc-random](https://hackage.haskell.org/package/mwc-random).
+3. Monadic adapters for pure generators (providing a uniform monadic
+   interface to pure and monadic generators).
+4. Faster in all cases except one by more than x18 (N.B. x18 not 18%) and
+   some cases (depending on the type) faster by more than x1000 - see
+   below for benchmarks.
+5. Passes a large number of random number test suites:
+   * [dieharder](http://webhome.phy.duke.edu/~rgb/General/dieharder.php "venerable")
+   * [TestU01 (SmallCrush, Crush, BigCrush)](http://simul.iro.umontreal.ca/testu01/tu01.html "venerable")
+   * [PractRand](http://pracrand.sourceforge.net/ "active")
+   * [gjrand](http://gjrand.sourceforge.net/ "active")
+   * See [random-quality](https://github.com/tweag/random-quality)
+     for details on how to do this yourself.
+6. Better quality split as judged by these
+	[tests](https://www.cambridge.org/core/journals/journal-of-functional-programming/article/evaluation-of-splittable-pseudorandom-generators/3EBAA9F14939C5BB5560E32D1A132637). Again
+	see [random-quality](https://github.com/tweag/random-quality) for
+	details on how to do this yourself.
+7. Unbiased generation of ranges.
+8. Updated tests and benchmarks.
+9. [Continuous integration](https://travis-ci.org/github/idontgetoutmuch/random).
```

Comment:
Should be haskell/random?

Reply:
Yes. Changed. Also removed link to generated haddock, since it will be useless once this version is released. I think it was mostly used for the proposal.